scripts/populate-session-tokens.js

```diff
@@ -80,11 +103,22 @@ if (count > 0) {
         uaOS: 'baz',
         uaOSVersion: 'qux',
         uaDeviceType: uaDeviceType
+      }).then(function () {
+        return sessionTokenId
       })
     }
 
     function hex (length) {
-      return Buffer(crypto.randomBytes(length).toString('hex'), 'hex')
+      return crypto.randomBytes(length)
+    }
+
+    function createDevice (uid, sessionTokenId, uaDeviceType) {
+      return db.createDevice(uid, hex(16), {
+        sessionTokenId: sessionTokenId,
+        name: 'fake device name',
+        type: uaDeviceType,
+        createdAt: Date.now()
+      })
     }
   })
```

Comment:
On first run of the script I had an incorrect patch level which it reported, bu left the processing around. Could add a catch on `db.connect` and do a `process.exit`.